src/lib/api.js

```diff
@@ -64,3 +64,11 @@ export const updatePurchaseTimeDB = async (token, item, state, estimate) => {
     await updateDoc(listRef, { items: arrayRemove(itemFinded) });
   }
 };
+
+export const deleteItems = async (token, itemFinded) => {
```

Comment:
Pequeño detalle: según entiendo esta función tiene el propósito de eliminar **1 item**, no varios items. ¿verdad, o me equivoco?

```suggestion
export const deleteItem = async (token, itemToDelete) => {
```

Reply:
Si!, solo se elimina 1 producto a traves de su boton deleted

Reply:
Gracias ale, arreglamos el nombre de la funcion ***f9373e3aeb2730c8ca44f084cb0d2ed5e1c3fb93***